utils/config.ts

```diff
@@ -28,7 +30,10 @@ export const JWTConfig = t.intersection([
     EXCHANGE_JWT_ISSUER: NonEmptyString,
     EXCHANGE_JWT_PUB_KEY: NonEmptyString,
     HUB_SPID_LOGIN_JWT_ISSUER: NonEmptyString,
-    HUB_SPID_LOGIN_JWT_PUB_KEY: NonEmptyString
+    HUB_SPID_LOGIN_JWT_PUB_KEY: NonEmptyString,
+    MAGIC_LINK_JWE_ISSUER: NonEmptyString,
+    MAGIC_LINK_JWE_PRIVATE_KEY: NonEmptyString,
```

Comment:
Add a mechanism that support the key rotation, with the primary key used for encoding and decoding JWE and the secondary only for decoding.

Reply:
Agree, but we can postpone key rotation (both for JWE and JWT) to another task, anyway.